prog/postgres/postgres_server_nexus.rb

```diff
@@ -447,15 +447,15 @@ def switch_to_new_timeline
   end
 
   label def taking_over
+    when_promote_set? do
+      postgres_server.update(synchronization_status: "ready")
+      decr_promote
```

Comment:
How about we move `decr_promote` to the wait label and `synchronization_status` update to `Succeeded` branch below. Then we can remove `when_promote_set` branch?

Reply:
Yeah, that makes sense, updated it.